### DISCUSSION TITLE
Calendar Icon request

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#983] [Request: Icon] [answered]
New Debugger design needs an icon that represent a Calendar. So that customers can select the custom range for applications' errors and warnings. 
 
Link from the Streamline: 
![image](https://user-images.githubusercontent.com/67388401/100777164-4c2ec800-33ba-11eb-84cb-1b37ebbb663b.png)

**Comment by serifluous** [comment]
Added to our backlog 👍 

It'll be out before the end of our January release.

**Comment by corinne-nagel** [comment, marked as answer]
Hey @sang-yeo , the calendar icon was added here: https://paste.twilio.design/icons
